components/SubmissionForm.vue

```diff
@@ -1,163 +1,185 @@
             >
-        </div>
-        <span
-            class="mb-2 text-primary-text text-sm font-normal font-sans"
-        >{{ $t('submitPage.spokenLanguage1') }}</span>
-        <p
-            v-show="!isValidInput.primarySpokenLangauge.value"
-            class="text-error text-xs font-sans"
-        >
-            {{ $t('submitPage.spokenLanguageValidation') }}
-        </p>
-        <select
-            v-model="submissionStore.selectLanguage1"
-            data-testid="submit-select-language1"
-            class="mb-5 px-3 py-3.5 w-96 h-12 bg-secondary-bg rounded-lg border border-primary-text-muted
-                    text-primary-text text-sm font-normal font-sans placeholder-primary-text-muted"
-            :placeholder="$t('submitPage.selectLanguage1')"
-            @change="initialValidationCheck(submissionStore.selectLanguage1, 'primaryLanguage')"
-        >
-            <option
-                v-for="(locale, index) in localeStore.localeDisplayOptions"
-                :key="index"
-                selected
-                :value="locale.code"
+                {{ $t('submitPage.spokenLanguageValidation') }}
+            </p>
+            <select
+                v-model="selectLanguage1"
+                data-testid="submit-select-language1"
+                class="mb-5 px-3 py-3.5 w-96 h-12 bg-secondary-bg rounded-lg border border-primary-text-muted
+                        text-primary-text text-sm font-normal font-sans placeholder-primary-text-muted"
+                :placeholder="$t('submitPage.selectLanguage1')"
+                @change="initialValidationCheck(selectLanguage1, 'primaryLanguage')"
             >
-                {{ locale.displayText }}
-            </option>
-        </select>
-        <span
-            class="mb-2 text-primary-text text-sm font-normal font-sans"
-        >{{ $t('submitPage.spokenLanguage2') }}</span>
-        <p
-            v-show="!isValidInput.secondarySpokenLanguage.value"
-            class="text-error text-xs font-sans"
-        >
-            {{ $t('submitPage.invalidOption') }}
-        </p>
-        <select
-            v-model="submissionStore.selectLanguage2"
-            data-testid="submit-select-language2"
-            class="mb-5 px-3 py-3.5 w-96 h-12 bg-primary-text-inverted rounded-lg border border-primary-text-muted
-                    text-primary-text text-sm font-normal font-sans placeholder-primary-text-muted"
-            :placeholder="$t('submitPage.selectLanguage2')"
-            @change="initialValidationCheck(submissionStore.selectLanguage1, 'secondaryLanguage')"
-        >
-            <option
-                v-for="(locale, index) in localeStore.localeDisplayOptions"
-                :key="index"
-                selected
-                :value="locale.code"
+                <option
+                    v-for="(locale, index) in localeStore.localeDisplayOptions"
+                    :key="index"
+                    selected
```

Comment:
double check this. I'm pretty sure this is making EVERY option selected. you might want to just have the first one in the array as selected or have none of them.

Reply:
yea it isn't necessary